app/code/Magento/Eav/Test/Unit/Model/TypeLocator/SimpleTypeTest.php

```diff
@@ -0,0 +1,139 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+namespace Magento\Eav\Test\Unit\Model\TypeLocator;
+
+use Magento\Eav\Model\TypeLocator\SimpleType;
+use Magento\Eav\Model\AttributeRepository;
+use Magento\Framework\Webapi\CustomAttribute\ServiceTypeListInterface;
+use Magento\Framework\TestFramework\Unit\Helper\ObjectManager;
+use Magento\Catalog\Model\ResourceModel\Eav\Attribute;
+use Magento\Framework\Reflection\TypeProcessor;
+
+/**
+ * Class SimpleTypeTest
+ *
+ * @package Magento\Eav\Test\Unit\Model\TypeLocator
+ */
+class SimpleTypeTest extends \PHPUnit\Framework\TestCase
+{
+    /**
+     * @var ObjectManager
+     */
+    private $objectManger;
+
+    protected function setUp()
+    {
+        $this->objectManger = new ObjectManager($this);
+    }
+
+    /**
+     * Test testGetType method
+     *
+     * @param ServiceTypeListInterface|\PHPUnit_Framework_MockObject_MockObject $serviceTypeListMock
+     * @param string $getFrontendInputReturn
+     * @param string $getBackendTypeReturn
+     * @param string $expected
+     * @dataProvider getTypeDataProvider
+     */
+    public function testGetType($serviceTypeListMock, $getFrontendInputReturn, $getBackendTypeReturn, $expected)
+    {
+        /**
+         * @var Attribute|\PHPUnit_Framework_MockObject_MockObject
+         */
+        $attributeMock = $this->getMockBuilder(Attribute::class)
```

Comment:
Please fix formatting and use `createMock` instead to make mock declarations more compact.